app/src/main/java/com/example/henzoshimada/feeltrip/FeelTripApplication.java

```diff
@@ -59,4 +65,79 @@ public static Participant getParticipant(){
         return participant;
     }
 
+    private static FilterController filterController = null;
+    public static FilterController getFilterController(){
+        if (filterController == null){
+            filterController = new FilterController();
+        }
+        return filterController;
```

Comment:
Anything we store in FeelTripApplication.java can be called from wherever, it's the handler for our entire app context.